src/Settings/Settings.ts

```diff
@@ -72,6 +72,8 @@ export const Settings = {
   ExcludeRunningScriptsFromSave: false,
   /**  Whether the game's sidebar is opened. */
   IsSidebarOpened: true,
+  /** Tail rendering intervall in ms */
+  TailRenderIntervall: 1000,
```

Comment:
```suggestion
  TailRenderInterval: 1000,
```
Also anywhere else I missed, you made this misspelling several times.